src/component.rs

```diff
@@ -105,6 +122,30 @@ impl<'a> Component<'a> {
     }
 }
 
+impl<'a> From<&'a Package> for Component<'a> {
+    fn from(package: &'a Package) -> Self {
+        Self {
+            component_type: ComponentType::Library,
+            scope: Some(Scope::Required),
+            metadata: ComponentCommon::from(package),
+            licenses: Some(vec![License::from(package)]),
```

Comment:
Does this handle dual-licensed code? I know that a large percent of Rust projects are MIT + Apache 2.0, so the single `License` made me curious how that fits into ClycloneDX

Reply:
More or less, the way we are using licenses is creating an `expression` field, so yeah, it'll take dual licensed etc... stuff

Reply:
We use `Vec` here because the actual spec allows for multiple licenses, but cargo only really provides us with that `expression`, so it becomes a Vec of 1, more or less